elk/evaluation/evaluate.py

```diff
@@ -20,8 +20,12 @@ class Eval(Run):
     def __post_init__(self):
         assert self.source, "Must specify a source experiment."
 
-        transfer_dir = elk_reporter_dir() / self.source / "transfer_eval"
-        self.out_dir = transfer_dir / "+".join(self.data.prompts.datasets)
+        # Set the output directory to the transfer directory if it's not specified
+        self.out_dir = (
+            transfer_eval_directory(self.source)
+            if self.out_dir is None
+            else self.out_dir
+        )
```

Comment:
previously was wrongly specified to be `transfer_dir / "+".join(self.data.prompts.datasets)`. 
Also i only set it if it wasn't already specified. Seems like sweep wants to specify their own directory so I didn't want to always override it.